### ISSUE TITLE
Theme_economist not working with patchwork?

### ISSUE THREAD

**Issue by heoa** [thomasp85/patchwork#113]
I am trying to get the theme economist to work with patchwork below.

**Example ggplot with theme economist**

[![enter image description here][2]][2]

**Example where theme economist not working**

[![enter image description here][1]][1]

**Minimal working example with patchwork where theme economist not working**

```
library(patchwork)
library(ggplot2)
library(ggthemes)

g1_ <- ggplot(data=mtcars, aes(x=hp,y=mpg)) + geom_point() #+ theme_economist() + scale_color_economist()
g2_ <- ggplot(data=mtcars,aes(x=hp,y=mpg)) + geom_point() + geom_smooth() + facet_wrap('gear') #+ theme_economist() + scale_color_economist()
g3_ <- ggplot(data=mtcars,aes(x=hp,y=mpg)) + geom_point() + geom_smooth() + facet_wrap('am')

#LET's add theme to this
(g1_ | g2_) / g1_ 

#THEME BW WORKNIG
(g1_ | g2_) / g3_  * theme_bw()
(g1_ | g2_) / g3_  & theme_bw()

#THEME ECONOMIST NOT WORKING WITH PATCHWORK
(g1_ | g2_) / g3_  * theme_economist() #* scale_color_economist()
(g1_ | g2_) / g3_  & theme_economist() #* scale_color_economist()

#THEME ECONOMIST
g3_  * theme_economist() * scale_color_economist()
g3_ + theme_economist() + scale_color_economist()
```

  [1]: https://i.stack.imgur.com/eH1td.png

  [2]: https://i.stack.imgur.com/F93mw.png

**Comment by steveharoz** [comment]
Patchwork currently drops the background, so the light blue is getting lost. A workaround would be to set the background color using PR #106. `  + plot_theme(background = "#d5e4eb")`

But keep in mind that that PR probably won't be the final syntax.

**Comment by thomasp85** [comment]
closed in lieu of #119
